app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
Ok. I am not sure about creating a module to extend for the purpose of loading an yaml file. What do you think @margOnline & @willhall88?